web/server/vue-cli/src/assets/userguide/userguide.md

```diff
@@ -334,133 +427,356 @@ stored.
-- [Report navigation tree](#userguide-report-navigation-tree)
-- [Button pane](#userguide-button-pane)
-- [Bug path view](#userguide-bug-path-view)
-- [Comments](#userguide-comment)
+Each line should begin with a `+` or a `-`, followed by a path glob pattern:
+- `+`: *ONLY* results from the matching file paths will be listed.
+- `-`: results from the matching file paths will not be listed.
 
-![Report view](images/report_view.png)
+For more information [see](https://github.com/Ericsson/codechecker/blob/master/docs/web/user_guide.md#format-of-component-file).
 
-## <a name="userguide-report-navigation-tree"></a> Report navigation tree
-Report Navigation Tree shows the found reports at the currently opened file. The
-reports are grouped by the severity level.
-You can navigate between them by clicking on a node in the tree.
+# Report details
+## Report Navigation Tree
+Report Navigation Tree shows the found reports at the currently opened file.
+The reports are grouped by the severity level. You can navigate between them
+by clicking on a node in the tree.
 
-![Report navigation tree](images/report_navigation_tree.png)
+![Report navigation tree](images/report/report_navigation_tree.png)
 
-## <a name="userguide-button-pane"></a> Button pane
+## Button pane
 Button Pane contains several items which help you to change or get some
-property of the currently opened report. 
-![Button pane](images/button_pane.png)
+property of the currently opened report.
+
+![Button pane](images/report/button_pane.png)
+
+### Report info
+`REPORT INFO` button shows detail information about the currently opened report
+such as *Report ID*, *Run name*, *Detection / Fix date* etc.
 
-### <a name="userguide-show-documentation"></a> Show documentation
-Show Documentation button shows the documentation of the actual checker which
+### Show documentation
+`SHOW DOCUMENTATION` button shows the documentation of the actual checker which
 identified by the currently opened report.
-![Checker documentation](images/checker_documentation.png)
 
-### <a name="userguide-change-review-status"></a> Change review status
+![Checker documentation](images/report/checker_documentation.png)
+
+### Change review status
 Reports can be assigned a review status of *Unreviewed*, *Confirmed*, *False
-positive*, *Intentional*, along with an optional comment on why this status was
-applied. We can change the review status from the default
-<span class="customIcon review-status-unreviewed"></span> _Unreviewed_ option
+positive*, *Intentional* along with an optional comment on why this status was
+applied.
+
+It can be changed on the GUI or by using
+[source code comments](https://github.com/Ericsson/codechecker/blob/master/docs/analyzer/user_guide.md#source-code-comments)
+(*codechecker_false_positive*, *codechecker_confirmed*, etc.)
+
+**Note**: source code comment is stronger and can overwrite the value in the
+database.
+
+We can change the review status on the GUI from the default
```

Comment:
This paragraph should go before the previous one because in that "Note" we describe the priorities between these two review status setting options.